BitmapFont/Rect_Test.cpp

```diff
@@ -6,73 +6,71 @@ namespace bmf
 {
 	TEST_CASE("Rect works properly", "[BitmapFontCache]")
 	{
-		SECTION("Create rect")
-		{
-			int a;
-			Rect newRect(0, 0, 1024, 512);
-			REQUIRE(newRect.width() == 1024);
-			REQUIRE(newRect.height() == 512);
-			REQUIRE(newRect.left() == 0);
-			REQUIRE(newRect.top() == 0);
+		// SECTION("Create rect")
+		// {
+			// Rect newRect(0, 0, 1024, 512);
+			// REQUIRE(newRect.width() == 1024);
+			// REQUIRE(newRect.height() == 512);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this block of commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=cpp%3ACommentedCode)